forge/csrc/passes/pre_placer_forge_passes.cpp

```diff
@@ -1,1290 +0,0 @@
-            broadcast->set_output_df(input->output_df());
-            consteval_graph->promote_node(std::move(broadcast));
-            current_shape = shape;
-        }
-    };
-
-    for (graphlib::Node *node : graph->nodes())
-    {
-        graphlib::ConstantInputNode *potential_input = dynamic_cast<graphlib::ConstantInputNode *>(node);
-        if (not potential_input)
-            continue;
-
-        // if there are multiple users they each might need a different set of broadcasts
-        // trivially just clone mutliple users
-        {
-            bool has_broadcast = false;
-            std::vector<graphlib::Edge> users = graph->user_data_edges(node);
-            for (graphlib::Edge const &edge : graph->user_data_edges(node))
-            {
-                auto edge_attr = graph->get_edge_attributes(edge);
-                for (auto const &tm : edge_attr->get_tms()) has_broadcast |= (tm.op == "broadcast");
-            }
-
-            if (not has_broadcast)
-                continue;
-
-            for (int i = 1; i < (int)users.size(); ++i)
-            {
-                graphlib::Edge edge = users[i];
-                auto edge_attr = graph->get_edge_attributes(edge);
-
-                auto input_clone = potential_input->clone();
-                input_clone->set_name(potential_input->name() + "_clone" + std::to_string(i));
-                auto graph_input = graph->add_node<graphlib::Node>(
-                    std::move(input_clone), graph->get_subgraph_id_for_node(edge.consumer_node_id));
-                graph->remove_edge(edge);
-                graph->add_edge(
-                    graphlib::Edge(
-                        graph_input->id(), 0, edge.consumer_node_id, edge.consumer_input_port_id, edge.edge_type),
-                    edge_attr);
-            }
-        }
-
-        for (graphlib::Edge const &edge : graph->user_data_edges(node))
-        {
-            graphlib::ConstantInputNode *input =
-                dynamic_cast<graphlib::ConstantInputNode *>(graph->node_by_id(edge.producer_node_id));
-            auto edge_attr = graph->get_edge_attributes(edge);
-            std::vector<graphlib::OpType> &tms = edge_attr->get_tms();
-
-            int index = 0;
-            std::vector<int> indices;
-            std::vector<graphlib::OpType> broadcasts;
-            for (auto const &tm : tms)
-            {
-                if (tm.op == "broadcast")
-                {
-                    indices.push_back(index);
-                    broadcasts.push_back(tm);
-                }
-                ++index;
-            }
-
-            if (broadcasts.empty())
-                continue;
-
-            for (auto iter = indices.rbegin(); iter != indices.rend(); ++iter) tms.erase(tms.begin() + *iter);
-
-            promote(graph, input, broadcasts);
-        }
-    }
-}
-
-// Convert TTForge graph to Forge graph
-std::unique_ptr<Graph> lower_to_forge_ops(Graph *graph)
```

Comment:
As lowering is removed, our lowering logic from op definition is redundant as well. Can you open tech-dept issues to remove those? 

_Note: There is no need for you to push this logic now, let's just track it as TODO item, with a good second issues label :))_